GitUI/Translation/English.xlf

```diff
@@ -7452,6 +7452,22 @@ Do you want to use this custom merge script?</source>
         <source>OK</source>
         <target />
       </trans-unit>
+      <trans-unit id="lblFullHistory.Text">
+        <source>Full history</source>
```

Comment:
If adding, add now. I would revert the translation strings if this is added.

Reply:
`B` is used twice. Could be moved to `Y` of `Simplify by decoration`.

The mnemonics do not work well because the labels are located behind the checkboxes.

The `OK` button does not react on `Enter`.
But it seems to work via the Form (refer to d87d912bfa4fb3c27dc80e7308970e12574347bd):

![image](https://user-images.githubusercontent.com/36601201/192648607-400a2bff-e894-4857-86f7-c7b33d8cdee4.png)

Reply:
`M` and `F` are used twice, too.

Apparently I often click the text of a checkbox in order to un/tick it.
This does not work in this dialog at all.
Using the builtin checkbox label will also solve "The mnemonics do not work well because the labels are located behind the checkboxes."

Reply:
Changed the mnemonics and added Enter handling as suggested.
The separate labels removed (also changed IgnoreCase to trail) so they are keyboard accelerated and clickable. 
Labels before the checkbox do not work with either. That could be changed later (if someone has the interest), the translations would not need to be changed. But that is a bigger redesign.

Reply:
> Changed the mnemonics and added Enter handling as suggested.
> The separate labels removed (also changed IgnoreCase to trail) so they are keyboard accelerated and clickable.

Thank you!

> Labels before the checkbox do not work with either. That could be changed later (if someone has the interest), the translations would not need to be changed. But that is a bigger redesign.
Edit: labels before checkboxes requires adding tabindex to the controls, so it is doable but not tonight

I think it is usable as it is.